microutil/loading.py

```diff
@@ -160,6 +201,16 @@ def load_mm_frames(data_dir, glob_pattern=None, chunkby_dims=['C', 'Z']):
     chunks = np.expand_dims(chunks, tuple(range(-1, -len(chunkby_dims) - 3, -1)))
 
     d_data = da.block(chunks.tolist())
+
     x_data = xr.DataArray(da.block(chunks.tolist()), dims=group_dims + chunkby_dims + ['Y', 'X'])
 
+    with open(position_dirs[0] + "/metadata.txt") as f:
+        meta = json.load(f)
```

Comment:
What do you think about adding an escape hatch of just providing a path to the metadata file. LIke say pos0 is borked so you only want to load pos1-3, but as I understand it the metadata file only exists in the pos0 folder?

Reply:
the metadata should exist in each position directory. so in practice this just uses the first one not necessarily pos0.

Maybe the move is to have a kwarg in the loading files that allow you to pass a dict directly. So if the metadata is borked you just make the dict yourself and zoop it through.